pkg/query-service/app/http_handler.go

```diff
@@ -3106,6 +3106,148 @@ func (aH *APIHandler) getSpanKeysV3(ctx context.Context, queryRangeParams *v3.Qu
+				Key:      "httpUrl",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["db.system"] = v3.AttributeKey{
+				Key:      "dbSystem",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["db.name"] = v3.AttributeKey{
+				Key:      "dbName",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["db.operation"] = v3.AttributeKey{
+				Key:      "dbOperation",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["peer.service"] = v3.AttributeKey{
+				Key:      "peerService",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["url.full"] = v3.AttributeKey{
+				Key:      "httpUrl",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.response.status_code"] = v3.AttributeKey{
+				Key:      "responseStatusCode",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.route"] = v3.AttributeKey{
+				Key:      "httpRoute",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.method"] = v3.AttributeKey{
+				Key:      "httpMethod",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.request.method"] = v3.AttributeKey{
+				Key:      "httpMethod",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.status_code"] = v3.AttributeKey{
+				Key:      "responseStatusCode",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.host"] = v3.AttributeKey{
+				Key:      "httpHost",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.host"] = v3.AttributeKey{
+				Key:      "httpHost",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["http.request.header.host"] = v3.AttributeKey{
+				Key:      "httpHost",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["server.address"] = v3.AttributeKey{
+				Key:      "httpHost",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
+			spanKeys["client.address"] = v3.AttributeKey{
+				Key:      "httpHost",
+				Type:     v3.AttributeKeyTypeTag,
+				IsColumn: true,
+				DataType: v3.AttributeKeyDataTypeString,
+			}
```

Comment:
Since the equivalent change in collect is recent and not released yet, we either have to hold this PR for enough data to backfill or just comment out these new additions so users don't see incorrect results for longer time ranges.